src/icons/badges/MetadataDefaultBadge.js

```diff
@@ -19,10 +20,10 @@ const MetadataDefaultBadge = ({ className = '', height = 16, title, width = 16 }
         width={width}
     >
         <g fill="none" fillRule="evenodd">
-            <circle cx="8" cy="8" fill="none" r="7.5" stroke="#767676" />
+            <circle cx="8" cy="8" fill="none" r="7.5" stroke={bdlNeutral02} />
             <path
                 d="M5.126 10.545a.57.57 0 0 1-.693.437.603.603 0 0 1-.416-.728l1.143-4.8c.122-.513.772-.62 1.039-.172L8 8.308l1.801-3.026c.267-.449.917-.34 1.039.173l1.143 4.8a.603.603 0 0 1-.416.727.57.57 0 0 1-.693-.437l-.817-3.43-1.572 2.643a.557.557 0 0 1-.97 0L5.943 7.116l-.817 3.43z"
-                fill="#767676"
+                fill={bdlNeutral02}
```

Comment:
Do not use neutrals, use bdlGray

Reply:
See https://opensource.box.com/box-ui-elements/#/Colors for available colors

Reply:
I am not seeing that variable name in BUI. Where could I find this variable?